### PR TITLE
Memsql

### PR DESCRIPTION
**Pull request by aouyang1**
Added memsql to installations. Also added in Apache License v2.0

### CODE REVIEW

**Comment by aouyang1** [comment]
test this please

**Comment by aouyang1** [comment]
test this please

File: Dockerfile [hunk 1]
```diff
@@ -11,10 +11,15 @@ RUN apt-get update \
     && apt-get install -y openssh-client \
     && apt-get install -y python \
     && apt-get install -y python-dev \
-    && apt-get install -y python-pip
+    && apt-get install -y python-pip \
+    && apt-get install -y git
 
 RUN pip install awscli
 
+RUN git clone https://github.com/sstephenson/bats.git /root/bats
```
Comment (ronaknnathani) [inline Dockerfile]: @aouyang1 this is pretty cool!

File: install/memsql/install_memsql_cluster.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+PEG_ROOT=$(dirname ${BASH_SOURCE})/../..
+source ${PEG_ROOT}/util.sh
+
+CLUSTER_NAME=$1
+
+MASTER_DNS=$(fetch_cluster_master_public_dns ${CLUSTER_NAME})
+
+script="${PEG_ROOT}/install/memsql/install_memsql.sh"
+run_script_on_node ${MASTER_DNS} ${script}
```
Comment (ronaknnathani) [inline install/memsql/install_memsql_cluster.sh]: @aouyang1 is it a single node installation?

Comment (aouyang1) [inline install/memsql/install_memsql_cluster.sh]: That's right. You then use the GUI to finish the deployment. It's quite intuitive. I give instructions after you install it on the single node

Comment (ronaknnathani) [inline install/memsql/install_memsql_cluster.sh]: @aouyang1 in that case, everything else looks good. Like the license headers!

**Pull request merged**